common/live-streaming.md

```diff
@@ -84,3 +85,25 @@ For custom apps, you can enable the live streaming destination manually:
 1. Ensure that you have roles for the broadcaster (who can publish their audio/video) and the viewer (who cannot publish audio/video)
 
 ### SDK integration
+
+Use the 100ms client-side SDKs to integrate streaming in your application. See code snippets for the web SDK [here](./../features/hls).
+
+### HLS playback for the audience
+
+![HLS playback](/docs/docs/v2/hls-playback.png)
+
+You can serve HLS streaming video from a [CDN](https://www.stackpath.com/edge-academy/what-is-a-cdn/) for live playback for the audience.
```

Comment:
I don't think the introduction of the CDN term here is useful. What's important is to communicate the difference between playback with the 100ms client SDK and without it

Reply:
fixed it.